### ISSUE TITLE
How to get nested ValuObjects changes

### ISSUE THREAD

**Issue by floresek** [javers/javers#216]
Hi,

I have classes:

```java
class Master {
    @Id
    String id;

    Detail detail;

    public Master(String id) {
        this.id = id;
    }
}

@ValueObject
class Detail {
    String data;
    Detail subdetail;

    public Detail(String data) {
        this.data = data;
    }
}
```

and test code:
```java
        Master master = new Master("1");
        master.detail = new Detail("detail-1");
        master.detail.subdetail = new Detail("subdetail-1.1");
        master.detail.subdetail.subdetail = new Detail("subdetail-1.1.1");
        javers.commit("anonymous", master);
```
I know how to read first level ValueObject changes:
```java
        List<Change> changes = javers
                .findChanges(QueryBuilder.byValueObjectId("1", Master.class, "detail")
                        .withNewObjectChanges(true)
                        .build());
```
but how to read second, third, ... - any level VO changes/snapshots?

f.e.
```java
        List<Change> changes = javers
                .findChanges(QueryBuilder.byValueObjectId("1", Master.class, "detail/subdetail")
                        .withNewObjectChanges(true)
                        .build());
```

**Comment by bartoszwalacik** [comment]
well, nested ValueObjects dont have a goood support yet :(

**Comment by bartoszwalacik** [comment]
related to https://github.com/javers/javers/issues/220

**Comment by bartoszwalacik** [comment]
example query

```
      def changes = javers.findChanges(QueryBuilder.byValueObjectId(1, DummyUserDetails,
              "dummyAddress/networkAddress").build())
```

**Comment by bartoszwalacik** [comment]
fixes merged to master, waiting for release

**Comment by bartoszwalacik** [comment]
released in 1.6.2
